vault-core/src/main/java/org/apache/jackrabbit/vault/fs/impl/io/DocViewImporter.java

```diff
@@ -990,13 +991,20 @@ private void removeReferences(@NotNull Node node) throws ReferentialIntegrityExc
         Optional<String> identifier = ni.getIdentifier();
         // try to set uuid via sysview import if it differs from existing one
         if (identifier.isPresent() && !node.getIdentifier().equals(identifier.get()) && !"rep:root".equals(ni.getPrimaryType().orElse(""))) {
+            long startTime = System.currentTimeMillis();
+            String previousIdentifier = node.getIdentifier();
+            log.info("Node stashing for {} starting, existing identifier: {}, new identifier: {}, import mode: {}",
```

Comment:
With info level that is too verbose. Can we lower to debug?

Reply:
Yes.

Reply:
FWIW, there's also

---
        for (String referencePath : removableReferencePaths) {
            log.info("Remove reference towards {} at {}", node.getIdentifier(), referencePath);
            session.removeItem(referencePath);
        }
---